### PR TITLE
Fix-SWP06 - Lack of Inputs Validation 

### PR DESCRIPTION
**Pull request by Ashuaidehao**

### CODE REVIEW

File: Swap/flamingo-contract-swap/FlamingoSwapPairWhiteList/FlamingoSwapPairWhiteList.cs [hunk 1]
```diff
@@ -28,6 +28,7 @@ public partial class FlamingoSwapPairWhiteList : SmartContract
         /// <returns></returns>
         public static bool AddRouter(UInt160 router)
         {
+            Assert(router.IsValid, "Invalid Address");
```
Comment (shargon) [inline Swap/flamingo-contract-swap/FlamingoSwapPairWhiteList/FlamingoSwapPairWhiteList.cs:31]: Remember that `IsValid` allow `Zero`

File: Swap/flamingo-contract-swap/FlamingoSwapRouter/FlamingoSwapRouterContract.Helper.cs [hunk 2]
```diff
@@ -48,6 +48,7 @@ private static void Assert(bool condition, string message, params object[] data)
         /// <returns></returns>
         public static UInt160 GetExchangePairWithAssert(UInt160 tokenA, UInt160 tokenB)
         {
+            Assert(tokenA.IsValid && tokenB.IsValid, "Invalid A or B Address");
```
Comment (shargon) [inline Swap/flamingo-contract-swap/FlamingoSwapRouter/FlamingoSwapRouterContract.Helper.cs:51]: Remember that `IsValid` allow `Zero`

File: Swap/flamingo-contract-swap/FlamingoSwapRouter/FlamingoSwapRouterContract.Admin.cs [hunk 3]
```diff
@@ -48,6 +48,7 @@ public static UInt160 GetAdmin()
         /// <returns></returns>
         public static bool SetAdmin(UInt160 admin)
         {
+            Assert(admin.IsValid && !admin.IsZero, "Invalid Address");
```
Comment (shargon) [inline Swap/flamingo-contract-swap/FlamingoSwapRouter/FlamingoSwapRouterContract.Admin.cs:51]: Remember that `IsValid` allow `Zero`

File: Swap/flamingo-contract-swap/FlamingoSwapPairWhiteList/FlamingoSwapPairWhiteList.cs (hunk 1 above)
Comment (Ashuaidehao) [reply Swap/flamingo-contract-swap/FlamingoSwapPairWhiteList/FlamingoSwapPairWhiteList.cs:31]: Input `Zero` won't influence whitelist's work.

File: Swap/flamingo-contract-swap/FlamingoSwapRouter/FlamingoSwapRouterContract.Helper.cs (hunk 2 above)
Comment (Ashuaidehao) [reply Swap/flamingo-contract-swap/FlamingoSwapRouter/FlamingoSwapRouterContract.Helper.cs:51]: Input `Zero` will get failure in next line, no need extra check here.

**Review by shargon — APPROVED** [review]

**Pull request merged**